fastlane/spec/helper/xcodes_helper_spec.rb

```diff
@@ -0,0 +1,66 @@
+describe Fastlane::Helper::XcodesHelper do
+  describe "#read_xcode_version_file" do
+    let(:xcode_version_path) { ".xcode-version" }
+
+    context ".xcode-version file exists" do
+      before do
+        allow(Dir).to receive(:glob).with(".xcode-version").and_return([xcode_version_path])
+        allow(File).to receive(:read).with(xcode_version_path).and_return("14")
+      end
+
+      it "return its content" do
+        expect(subject.class.read_xcode_version_file).to eq("14")
+      end
+    end
+
+    context "no .xcode-version file exists" do
+      before do
+        allow(Dir).to receive(:glob).with(".xcode-version").and_return([])
+      end
+
+      it "return nil" do
+        expect(subject.class.read_xcode_version_file).to eq(nil)
+      end
+    end
+  end
+
+  describe "#find_xcodes_binary_path" do
+    it "returns the binary path when it's installed" do
+      expect(subject.class).to receive(:find_xcodes_binary_path).and_return("/opt/homebrew/bin/xcodes")
+      expect(subject.class.find_xcodes_binary_path).to eq("/opt/homebrew/bin/xcodes")
```

Comment:
```suggestion
      expect(subject.class).to receive(:find_xcodes_binary_path).and_return("/path/to/bin/xcodes")
      expect(subject.class.find_xcodes_binary_path).to eq("/path/to/bin/xcodes")
```

for consistency with [xcodes_spec.rb](https://github.com/fastlane/fastlane/pull/20672/files#diff-2ca850ff6a7654f270c5deb87cbb737659ee2ee968e955924ae6524b170b78b5R6)?

Reply:
Oops, good catch! Left this behind by mistake after doing some local tests 😬